modules/functional-test/src/main/java/com/intuit/wasabi/tests/perftest/PerfTest.java

```diff
@@ -0,0 +1,127 @@
+/*******************************************************************************
+ * Copyright 2016 Intuit
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *******************************************************************************/
+package com.intuit.wasabi.tests.perftest;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import com.intuit.wasabi.tests.library.TestBase;
+import com.intuit.wasabi.tests.library.util.Constants;
+import com.intuit.wasabi.tests.model.Application;
+import com.intuit.wasabi.tests.model.Bucket;
+import com.intuit.wasabi.tests.model.Experiment;
+import com.intuit.wasabi.tests.model.factory.ApplicationFactory;
+import com.intuit.wasabi.tests.model.factory.BucketFactory;
+import com.intuit.wasabi.tests.model.factory.ExperimentFactory;
+
+public class PerfTest extends TestBase {
+
+    // lists that holds the experiments of test applications
+    List<Experiment> experimentListOfCTG = new ArrayList<Experiment>();
+    List<Experiment> experimentListOfQBO = new ArrayList<Experiment>();
+
+    // create performance test Applications
+    Application ctgApplication = ApplicationFactory.createApplication().setName("wasabiloadtest_CTG");
```

Comment:
Can we move the application name to a string constant at the top of the class? It increases readability when we want to refer to these literals to use elsewhere.